fluent-react/src/localized.js

```diff
@@ -84,6 +83,8 @@ export default class Localized extends Component {
     const { id, attrs, children } = this.props;
     const elem = Children.only(children);
 
+    const markupParser = this.context.parseMarkup || parseMarkup;
```

Comment:
With a `createParseMarkup` creator, this can become:

```js
const parseMarkup = this.context.parseMarkup || createParseMarkup;
```

Or even, using default values and destructuring:

```js
const {parseMarkup = createParseMarkup()} = this.context;
```

Reply:
(I'd prefer for the variable used throughout the file to be called `parseMarkup` rather than `markupParser`.)

Reply:
Actually, this isn't such a great idea :) We'd end up re-creating the default `parseMarkup` function for every `Localized`. Sorry for the confusion.

Reply:
You could just cache the function once created to avoid that (personally I'd just move the check for document being missing into Localized)

Reply:
I added caching :)